.github/workflows/deploy-bridgedigitalhealth-tst.yaml

```diff
@@ -0,0 +1,16 @@
+name: deploy-arc-tst
```

Comment:
```suggestion
name: deploy-open-bridge-staging
```

Reply:
Changing to tst (to test on the infra I already deployed).